apps/src/javalab/Javalab.js

```diff
@@ -61,8 +61,11 @@ Javalab.prototype.init = function(config) {
   this.skin = config.skin;
   this.level = config.level;
   this.channelId = config.channel;
-  // Pulls dark mode from user preferences
-  this.isDarkMode = !!config.usingDarkModePref;
+  // Pulls dark mode from user preferences; default to true if not set
+  this.isDarkMode =
+    config.usingDarkModePref === undefined ||
+    config.usingDarkModePref === null ||
+    config.usingDarkModePref;
```

Comment:
You mentioned this relies on a separate change to ensure this value can be false in the table. Has that already been merged? If yes, link it in the PR. If no, a useful convention we often use is chained PRs: 
staging <- persistUserPreferences <- darkmode
^ that allows the code reviews to happen separately, but prevents a well-meaning co-worker from merging a change without the prefactor that supports it.

Am I right in saying that if this is merged as-is, light mode will no longer work?

Reply:
Yep, I just posted that PR so that has not been yet merged. I'll add this as a comment in the description.

And yes correct, if this is merged as-is, then the light mode preference will no longer persist; users could still select light mode but that preference would not be saved.